pkg/api/controllers/fileshare.go

```diff
@@ -371,6 +371,35 @@ func (f *FileSharePortal) UpdateFileShare() {
 		return
 	}
 
+	if fshare.Name == "" {
```

Comment:
Name == "" is a valid case for update

Reply:
Two things need to take care:
1. If i want to update only description? [Currently it is not allowing, asking for name parameter also, which is not correct.] Description alone should be updated.
2. If you add this code, then it will create another problem. Always name should be passed even though we don't want to modify the name.